documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+#JavaScript SDK for Nether
+
+## Work in Progress
+
+## Downloading the JavaScript SDK for Nether
+
+### Installing the JavaScript SDK for Nether
+
+## Usage
+
+### Analytics
+
+#### Active sessions - daily
+You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
```

Comment:
@stuartleeks I was thinking about adding grouping in but as the API's weren't done this way I decided to leave as is. Think the grouping would cut down on the sdk methods, making the sdk look cleaner to the user